src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let i = 0;
+  const newRobot = {};
+
+  for (const each in robot) {
```

Comment:
The same. Name 'each' not meaningful. Chose a more suitable name for this variable (e.g. 'part', 'value' etc...)